### ISSUE TITLE
original data at [http://dmitriev.speciesfile.org/Export/insects.zip] no longer available

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/inhs-insects#2]
from Elton's access.tsv log on globi server, the last version of insects.zip was seen on 2021-06-19T01:25:01.491Z

```
...
globalbioticinteractions/inhs-insects	http://dmitriev.speciesfile.org/Export/insects.zip	ce4ba8b392b6c14a83ecdc4aa069209e5283cfdf05ad53a10c9cd7693c4c0ab0	2021-06-19T01:25:01.491Z	
...
```

Unfortunately, today (2021-06-24), original data at [http://dmitriev.speciesfile.org/Export/insects.zip] no longer seem to be available.

```
$ curl "http://dmitriev.speciesfile.org/Export/insects.zip" > /dev/null
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:02:09 --:--:--     0curl: (7) Failed to connect to dmitriev.speciesfile.org port 80: Connection timed out
```

@mjy et al. do you have any recommendation on what dcwa to use to access interaction data from INHS Entomology collections?

**Comment by tmcelrath** [comment]
@jhpoelen the link should work again now

**Comment by jhpoelen** [comment]
@tmcelrath thanks! Is that the most up-to-date source location for your collection?

**Comment by tmcelrath** [comment]
Yeah it is for now. It's now being indexed by GBIF and iDigBio again.
There are, a LOT more records now. But our export format is completely different so I'm curious to see how easily you can find the interactions. They should be all in the "Habitat" section for now, but we should be explicitly exporting interactions soon, right @mjy?

**Comment by jhpoelen** [comment]
@tmcelrath I'd be happy to take any format, not just DwC . Just need a publicly available resource.

**Comment by jhpoelen** [comment]
Can you please provide some examples - so I can verify that your data is indexed properly?

**Comment by tmcelrath** [comment]
[localized_to_INHS.csv](https://github.com/globalbioticinteractions/inhs-insects/files/9812356/localized_to_INHS.csv)

Hey @jhpoelen here is a sample list of 450+ occurrences that have "habitat" data that should be indexable. Right now in TaxonWorks we have these explicitly indexed as "localized to" some taxon, but that explicit association is not exported yet. So right now we just have the verbatim DWC "Habitat" strings. 

@mjy can give an estimate on when those explicit biological associations will be exported in our DWC dumps.

**Comment by jhpoelen** [comment]
@tmcelrath thanks for sharing the example. A quick spot check suggests that the examples you shared are not yet indexed in the INHS collection. 

Great to see that so much interaction data lies under the surface!

 Let me know when you are ready for me to update the index configuration for INHS. https://github.com/globalbioticinteractions/inhs-insects . Note that non-insect interactions are most welcome and we can update the configuration to reflect that.

**Comment by tmcelrath** [comment]
Non-insect interactions should definitely be added. Basically all arthropods should be indexed, although we have extremely few Crustacea. Is that what you mean?

**Comment by jhpoelen** [comment]
Yep, just opening the door to any kind of species interaction that you'd like others to discover in your collections.

**Comment by mjy** [comment]
Technically you have this now via the mega-crawl of the TW API I believe?

**Comment by jhpoelen** [comment]
@mjy I could not find evidence to suggest that 

> Poelen, Jorrit H. (2023). A biodiversity dataset graph: Biological Associations in TaxonWorks hash://sha256/a4d651aac5220487835e6178511886e98b845b2d98cb7c5447fb2b042e0654d2 hash://md5/849edbe55e31e54ea5cdaba0188c5655 (0.2) [Data set]. Zenodo. https://doi.org/10.5281/zenodo.8253729

includes all INHS Insect collection records.

For instance, INHS Insects appear to have (from https://depot.globalbioticinteractions.org/reviews/globalbioticinteractions/inhs-insects/)

Most Frequently Mentioned Primary Taxa (up to 20 most frequent)sourceTaxonName | count
-- | --
Mezira granulata (Say, 1832) | 258
Psyllidae Latreille, 1807 | 138
Pulicidae Billberg, 1820 | 125
Empoasca | 122
Orchopeas howardi (Baker, 1895) | 115

But, GloBI's index of TaxonWorks corpus yielded no hits on 


```
curl "https://depot.globalbioticinteractions.org/reviews/globalbioticinteractions/taxonworks/indexed-interactions.tsv"\
 | grep "Mezira granulata"\
 | wc -l
```

0 

I would have expected these interaction to appear. 

Can you verify whether taxonworks includes interactions associated with the previously obtained interaction in the INHS Insect collection review as attached ?

https://depot.globalbioticinteractions.org/reviews/globalbioticinteractions/inhs-insects/review.zip

[review.zip](https://github.com/globalbioticinteractions/inhs-insects/files/13494482/review.zip)

**Comment by jhpoelen** [comment]
Note that

```
curl -I http://dmitriev.speciesfile.org/Export/insects.zip
```

produces - 

``` 
HTTP/1.1 200 OK
Content-Length: 192034513
Content-Type: application/x-zip-compressed
Last-Modified: Fri, 02 Aug 2024 02:12:46 GMT
Accept-Ranges: bytes
ETag: "394c637781e4da1:0"
Server: Microsoft-IIS/7.5
X-Powered-By: ASP.NET
Date: Wed, 28 Jan 2026 16:15:47 GMT
```

suggesting that the resource is available.
